plugins/telemetry/telemetry.h

```diff
@@ -43,6 +43,21 @@ class Telemetry : public PluginBase {
         float relative_altitude_m; /**< @brief Altitude relative to takeoff altitude in metres */
     };
 
+    /**
+     * @brief Position and velocity local type.
```

Comment:
Please describe what it is for someone that doesn't know or understand PX4.

Reply:
resolved